CONTRIBUTING.md

```diff
@@ -2,12 +2,15 @@
 
 Thanks for taking the time to contribute!
 
-The following is a set of guidelines for contributing to zulip-electron. These are just guidelines, not rules, use your best judgment and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to zulip-electron. These are just guidelines, not rules, use your best judgement and feel free to propose changes to this document in a pull request.
 
 ## Getting Started
 
 Zulip-Desktop app is built on top of [Electron](http://electron.atom.io/). If you are new to Electron please head over to [this](http://jlord.us/essential-electron/) great article.
 
+## Chat
+Feel free to reach us out at [Zulip-Chat](https://chat.zulip.org) in the 'electron' section.
```

Comment:
Maybe make this link be written like the ones we have in  https://github.com/zulip/zulip/blob/master/README.md, where we link to both the docs on the community and the specific stream?

Also, it'd be great to add a link to Zulipbot here as well.